### ISSUE TITLE
sashimi group

### ISSUE THREAD

**Issue by Qingru44004** [Xinglab/rmats2sashimiplot#124]
Hi,

Does the sashimi plot support drawing more than 2 groups by customization? If not, is there any other way that I can do it? Thanks.

Best,
Qingru

**Comment by EricKutschera** [comment]
Yes, you can add more than 2 groups with `--group-info`. The example in the README could be modified to have three groups with grouping.gf changed to:
```
group1name: 1-2
group2name: 3-4
group3name: 5-6
```
https://github.com/Xinglab/rmats2sashimiplot/tree/v3.0.0?tab=readme-ov-file#using-a-group-file

**Comment by Qingru44004** [comment]
Thanks. If I have three groups of data. I did rmats on each pair. Now, I want to plot sashimi on the common events of these three groups. How should I do it?
